docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Build an Outlook add-in with a Teams manifest (preview)
+description: Learn how to build a simple Outlook task pane add-in with a JSON manifest.
+ms.date: 05/24/2022
+ms.prod: outlook
+ms.localizationpriority: high
+---
+
+# Build an Outlook add-in with a Teams manifest (preview)
+
+In this article, you'll walk through the process of building an Outlook task pane add-in that displays a property of a selected message, triggers a notification on the reading pane, and inserts text into a message on the compose pane. This add-in will use a preview version of the JSON-formatted manifest that Teams extensions, like custom tabs and messaging extensions, use. For more information about this manifest, see [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
```

Comment:
Consider either removing that bit I've removed about this being what Teams uses, or spinning it like a value-add and promoting the cross-app potential.
```suggestion
In this article, you'll walk through the process of building an Outlook task pane add-in that displays a property of a selected message, triggers a notification on the reading pane, and inserts text into a message on the compose pane. This add-in will use a preview version of the JSON-formatted manifest. For more information about this manifest, see [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
```